### PR TITLE
Port System.Security source comments to Docs

### PR DESCRIPTION
**Pull request by carlossanlop**
Automatically ported triple slash source code comments from System.Security.Claims and System.Security.Cryptography to dotnet-api-docs.

### CODE REVIEW

**Comment by carlossanlop** [comment]
Adding area owners  @bartonjs, @GrabYourPitchforks for review.

@rpetrusha @mairaw please take a look whenever you get a chance for language review.

**Comment by carlossanlop** [comment]
Apologies for all the spaces that got unexpectedly modified in the remarks. Let me try to fix that so the changes look cleaner.

File: xml/System.Security.Claims/ClaimsPrincipal.xml [hunk 1]
```diff
@@ -856,7 +856,13 @@ if (null != principal)
         <param name="match">The function that performs the matching logic.</param>
         <summary>Retrieves the first claim that is matched by the specified predicate.</summary>
         <returns>The first matching claim or <see langword="null" /> if no match is found.</returns>
-        <remarks></remarks>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+Each <xref:T:System.Security.Claims.ClaimsIdentity> is called. <see also cref="M:System.Security.Claims.ClaimsIdentity.FindFirst(System.String)" />.
```
Comment (carlossanlop) [inline xml/System.Security.Claims/ClaimsPrincipal.xml]: Question for @rpetrusha: This is the first time I detect a "see also" tag in the remarks, but in this case, it seems it should be a simple xref link to the definition, preceded by the plain text "see also". Would you agree?:

```suggestion
Each <xref:T:System.Security.Claims.ClaimsIdentity> is called. See also: <xref:"M:System.Security.Claims.ClaimsIdentity.FindFirst(System.String)>.
```

Comment (carlossanlop) [reply xml/System.Security.Claims/ClaimsPrincipal.xml]: Same question applies for the "see also" cases below.

File: xml/System.Security.Cryptography.Xml/RSAKeyValue.xml [hunk 2]
```diff
@@ -168,7 +168,13 @@
       <Docs>
         <summary>Returns the XML representation of the <see cref="T:System.Security.Cryptography.RSA" /> key clause.</summary>
         <returns>The XML representation of the <see cref="T:System.Security.Cryptography.RSA" /> key clause.</returns>
-        <remarks>To be added.</remarks>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+Based upon https://www.w3.org/TR/xmldsig-core/#sec-RSAKeyValue.
```
Comment (carlossanlop) [inline xml/System.Security.Cryptography.Xml/RSAKeyValue.xml]: @rpetrusha How should we show hyperlinks in remarks?

File: xml/System.Security.Claims/ClaimsPrincipal.xml (hunk 1 above)
Comment (rpetrusha) [reply xml/System.Security.Claims/ClaimsPrincipal.xml]: Yes, I agree, @carlossanlop. Incidentally, when the crefs are transformed to xrefs, the type/member identifier is retained when it shouldn't be. So `<xref:T:System.Security.Claims.ClaimsIdentity>` is an invalid xref; it should be `<xref:System.Security.Claims.ClaimsIdentity>`.

File: xml/System.Security.Cryptography.Xml/RSAKeyValue.xml (hunk 2 above)
Comment (rpetrusha) [reply xml/System.Security.Cryptography.Xml/RSAKeyValue.xml]: If there's a hyperlink with hyperlink text, the format is `[`*hyperlinktext*`](`*url*`)`.
If there's no separate hyperlink text, the format is `<`*url*`>`.

Comment (rpetrusha) [inline xml/System.Security.Cryptography.Xml/RSAKeyValue.xml]:
```suggestion
Based upon [The RSAKeyValue Element](https://www.w3.org/TR/xmldsig-core/#sec-RSAKeyValue) in the W3C Recommendation [XML Signature Syntax and Processing Version 1.1](https://www.w3.org/TR/xmldsig-core/).
```

File: xml/System.Security.Cryptography.Xml/RSAKeyValue.xml [hunk 3]
```diff
@@ -238,7 +244,13 @@
       <Docs>
         <param name="value">The XML element from which to load the <see cref="T:System.Security.Cryptography.RSA" /> key clause.</param>
         <summary>Loads an <see cref="T:System.Security.Cryptography.RSA" /> key clause from an XML element.</summary>
-        <remarks>To be added.</remarks>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+Based upon https://www.w3.org/TR/xmldsig-core/#sec-RSAKeyValue.
```
Comment (rpetrusha) [inline xml/System.Security.Cryptography.Xml/RSAKeyValue.xml]:
```suggestion
Based upon [The RSAKeyValue Element](https://www.w3.org/TR/xmldsig-core/#sec-RSAKeyValue) in the W3C Recommendation [XML Signature Syntax and Processing Version 1.1](https://www.w3.org/TR/xmldsig-core/).
```

File: xml/System.Security.Cryptography.Xml/RSAKeyValue.xml [hunk 4]
```diff
@@ -168,7 +168,13 @@
       <Docs>
         <summary>Returns the XML representation of the <see cref="T:System.Security.Cryptography.RSA" /> key clause.</summary>
         <returns>The XML representation of the <see cref="T:System.Security.Cryptography.RSA" /> key clause.</returns>
-        <remarks>To be added.</remarks>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+Based upon [The RSAKeyValue Element](https://www.w3.org/TR/xmldsig-core/#sec-RSAKeyValue) in the W3C Recommendation [XML Signature Syntax and Processing Version 1.1](https://www.w3.org/TR/xmldsig-core/).
```
Comment (bartonjs) [inline xml/System.Security.Cryptography.Xml/RSAKeyValue.xml]: It's actually based on the 1.0 version (https://www.w3.org/TR/2002/REC-xmldsig-core-20020212/ and https://www.w3.org/TR/2002/REC-xmldsig-core-20020212/#sec-RSAKeyValue).

Comment (rpetrusha) [reply xml/System.Security.Cryptography.Xml/RSAKeyValue.xml]: @carlossanlop, could you make this correction, and then I'll merge your PR?

Comment (carlossanlop) [inline xml/System.Security.Cryptography.Xml/RSAKeyValue.xml]:
```suggestion
Based upon [The RSAKeyValue Element](https://www.w3.org/TR/2002/REC-xmldsig-core-20020212/#sec-RSAKeyValue) in the W3C Recommendation [XML Signature Syntax and Processing Version 1.0](https://www.w3.org/TR/2002/REC-xmldsig-core-20020212).
```

File: xml/System.Security.Cryptography.Xml/RSAKeyValue.xml [hunk 5]
```diff
@@ -238,10 +244,16 @@
       <Docs>
         <param name="value">The XML element from which to load the <see cref="T:System.Security.Cryptography.RSA" /> key clause.</param>
         <summary>Loads an <see cref="T:System.Security.Cryptography.RSA" /> key clause from an XML element.</summary>
-        <remarks>To be added.</remarks>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+Based upon [The RSAKeyValue Element](https://www.w3.org/TR/xmldsig-core/#sec-RSAKeyValue) in the W3C Recommendation [XML Signature Syntax and Processing Version 1.1](https://www.w3.org/TR/xmldsig-core/).
```
Comment (carlossanlop) [inline xml/System.Security.Cryptography.Xml/RSAKeyValue.xml]: @bartonjs is this hyperlink also based in version 1.0, like in the other hyperlink you commented on (line 174)?

File: xml/System.Security.Cryptography.Xml/RSAKeyValue.xml (hunk 2 above)
Comment (carlossanlop) [reply xml/System.Security.Cryptography.Xml/RSAKeyValue.xml]: Thank you. This is outdated now.

File: xml/System.Security.Cryptography.Xml/RSAKeyValue.xml (hunk 5 above)
Comment (bartonjs) [reply xml/System.Security.Cryptography.Xml/RSAKeyValue.xml]: Yep, the whole namespace is based on v 1.0

Comment (carlossanlop) [reply xml/System.Security.Cryptography.Xml/RSAKeyValue.xml]:
```suggestion
Based upon [The RSAKeyValue Element](https://www.w3.org/TR/2002/REC-xmldsig-core-20020212/#sec-RSAKeyValue) in the W3C Recommendation [XML Signature Syntax and Processing Version 1.0](https://www.w3.org/TR/2002/REC-xmldsig-core-20020212).
```

Comment (carlossanlop) [reply xml/System.Security.Cryptography.Xml/RSAKeyValue.xml]: Ok done. Merged. Thank you.

File: xml/System.Security.Cryptography.Xml/RSAKeyValue.xml (hunk 4 above)
Comment (carlossanlop) [reply xml/System.Security.Cryptography.Xml/RSAKeyValue.xml]: The other one was 1.0 too. I updated it and merged it.

**Comment by carlossanlop** [comment]
@rpetrusha this is ready, we just need for the build to pass.

**Comment by rpetrusha** [comment]
Thanks, @carlossanlop. There was one remaining broken link, which I've fixed. I'll merge your PR now.

**Pull request merged**